R/utils.R

```diff
@@ -81,6 +81,45 @@ checkout <- function(branch, debug) {
   )
 }
 
+#' @title Running the node script "performance_test" is system-dependent
+#'
+#' @param project_path path to project directory (one level above node)
+#'
+#' @keywords internal
+performance_test_cmd <- function(project_path) {
+  if (grepl("win", .Platform$OS.type, ignore.case = TRUE)) {
+    glue("yarn --cwd \"{fs::path(project_path, 'node')}\" performance-test")
```

Comment:
Maybe it also works on linux. Did you try? Then we do not need the if here.

Reply:
Initially I had a problem with the different command, hence the platform-dependant. I'll check it again now that it has stabilized.

Reply:
I tried it and it fails on windows! I think it needs to be "&&"